packages/react-core/src/components/ProgressStepper/examples/ProgressStepperBasicWithDescription.tsx

```diff
@@ -15,7 +15,13 @@ export const ProgressStepperBasicWithDescription: React.FunctionComponent = () =
     <ProgressStep
       variant="info"
       isCurrent
-      description="This is the second thing to happen"
+      description={
```

Comment:
Three is no need to update the example.  If you want to add a test to verify the change that is fine.  I do not think that is necessary though.

Reply:
Sounds good! I'll revert the example back and add a test. Thanks @thatblindgeye and @tlabaj !!

Reply:
@tlabaj it appears that there's no way to test `<br />` through react testing according to [https://github.com/testing-library/dom-testing-library/issues/750](https://github.com/testing-library/dom-testing-library/issues/750)

Reply:
That is ok.  It is not really a necessary test.